.github/workflows/analysis.yml

```diff
@@ -13,12 +13,16 @@ jobs:
       - run:  sudo apt-get update
       - name: Install pylint
         run:  |
-          sudo apt-get install python3-setuptools
+          sudo apt-get install python3-setuptools ruby
           sudo pip3 install pylint beautifulsoup4 html5lib
+          sudo gem install mdl
+          sudo chmod go-w /usr/share/rust/.cargo/bin
```

Comment:
On startup, Ruby complain about a permission issue here - so this cleans that up in our output.